.husky/pre-commit.sh

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+/**
```

Shellcheck (0.10.0)

[error] Was this intended as a comment? Use # in sh.

(SC1127)